fmriprep/workflows/base.py

```diff
@@ -200,8 +200,8 @@ def init_single_subject_wf(subject_id: str):
 
     if not anat_derivatives and not subject_data['t1w']:
         raise Exception(
-            "No T1w images found for participant {}. "
-            "All workflows require T1w images.".format(subject_id)
+            f"No T1w images found for participant {subject_id}. "
+            f"All workflows require T1w images."
```

Comment:
I suspect this now fits on one line...

```suggestion
            f"No T1w images found for participant {subject_id}. All workflows require T1w images."
```